tutorials/plot_point_spread.py

```diff
@@ -0,0 +1,144 @@
+"""
+.. _point_spread:
+
+Corrupt known signal with point spread
+======================================
+
+The aim of this tutorial is to demonstrate how to put a known signal at a
+desired location(s) in a :class:`SourceEstimate` and then corrupt the signal
```

Comment:
I guess it needs to be:
```
:class:`mne.SourceEstimate`
```
I thought it would resolve without the `mne.` in front, but looking at the Circle build it does not.